### DISCUSSION TITLE
getData() and setData() refer to local, what's local?

### DISCUSSION THREAD

**Discussion by ApolloX939** [replit/kaboom#596] [Q&A] [answered]
Hi I developed a [web shooter](https://beanzilla.net/games/invaders_wars/) on my site.

For a upcoming "update" to my game I was thinking of supporting saving the player's score.

So to start I hit the Kaboomjs docs on all things kaboom, and saw getData() and setData().

My confusion is the docs refer to getData() and setData() as accessing local storage, but what does that mean? (server-side or client-side?)

If it's server-side I can make a global high-score board then, but if it's client-side I'd only track that client/player.

**Comment by lajbel** [comment, marked as answer]
`getData()` and `setData()` are local, you can storage things in the browser, for use multiplayer save of data, you can use Replit DB, Mongo, or like it.

**Reply by ApolloX939** [reply]
Ah, okay so it's kind of like a cookie then? stored client-side, Thank you.

**Reply by slmjkdbtl** [reply]
Yes it's on client side but not exactly like cookie, it's just a simple wrapper around [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage)
